i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-wsus-nsclient.md

```diff
@@ -1,129 +1,273 @@
 
-1. Depending on the monitoring method chosen (NRPE or RestAPI), install the relevant Centreon Plugin package on every Centreon
-Poller expected to monitor WSUS through *centreon-nsclient*:
+Quel que soit le type de la licence (*online* ou *offline*), installez le Pack **Microsoft WSUS**
+depuis l'interface web et le menu **Configuration > Plugin Packs > Gestionnaire**.
 
-* NRPE
+### Plugin
+
+Utilisez les commandes ci-dessous en fonction du gestionnaire de paquets de votre système d'exploitation :
+
+<Tabs groupId="sync">
+<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
 
 ```bash
-yum install centreon-nrpe-plugin
+dnf install centreon-nrpe3-plugin
 ```
 
-* RestAPI
+</TabItem>
+<TabItem value="CentOS 7" label="CentOS 7">
 
 ```bash
-yum install centreon-plugin-Operatingsystems-Windows-Restapi
+yum install centreon-nrpe3-plugin
 ```
 
-2. On the Central Server, install the Centreon Pack RPM:
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
 
 ```bash
-yum install centreon-pack-applications-wsus-nsclient
+apt install centreon-nrpe3-plugin
 ```
 
-3. On the Centreon Web interface, install the *Microsoft WSUS* Centreon Pack from the **Configuration > Plugin Packs > Manager** page
-
 </TabItem>
 </Tabs>
 
-## Host configuration
+## Configuration
 
-* On the Centreon Web Interface, go to **Configuration > Hosts** and add a new Host
-* Set the Host IP address and select the relevant Host template according to the monitoring method chosen:
-    * *App-Wsus-NRPE-custom* for NRPE
-    * *App-Wsus-NSClient-05-Restapi-custom* for RestAPI
-* Depending on the Host template, fill the Macro fields as follows:
+### Hôte
 
-<Tabs groupId="sync">
-<TabItem value="App-Wsus-NRPE-custom" label="App-Wsus-NRPE-custom">
+* Ajoutez un hôte à Centreon depuis la page **Configuration > Hôtes**.
+* Complétez les champs **Nom**, **Alias** & **IP Address/DNS** correspondant à votre serveur **WSUS Server**.
+* Appliquez le modèle d'hôte **App-Wsus-NRPE-custom**.
+* Une fois le modèle appliqué, les macros ci-dessous indiquées comme requises (**Obligatoire**) doivent être renseignées.
 
-| Mandatory | Name             | Description                                                                         |
-|:----------|:-----------------|:------------------------------------------------------------------------------------|
-| X         | NRPECLIENT       | NRPE Plugin binary to use (Default: 'check_centreon_nrpe')                          |
-| X         | NRPEPORT         | NRPE Port of the target server (Default: '5666')                                    |
-| X         | NRPETIMEOUT      | Timeout value (Default: '30')                                                       |
-|           | NRPEEXTRAOPTIONS | Any extra option you may want to add to every command\_line (Default: '-u -m 8192') |
+| Obligatoire | Macro                     | Description                                                                            |
```

Comment:
wouldn't it be clearer to distinguish the protocols with tabs (like the current version ?)
EN version is ok though